salt/states/zpool.py

```diff
@@ -30,6 +30,23 @@
               /dev/disk2
               /dev/disk3
 
+    partitionpool:
+      zpool.present:
+        - config:
+            import: false
+            force: true
+        - properties:
+            comment: disk partition salty storage pool
+            ashift: '12'
+            feature@lz4_compress: enabled
+        - filesystem_properties:
+            compression: lz4
+            atime: on
+            relatime: on
+        - layout:
+            disk-0:
```

Comment:
Yes, that worked!  So how would you like to proceed with this now?  Shall I remove the `disk-0` in a rebase (and then mess up all of the commit IDs) or shall I just append it as another commit?